### ISSUE TITLE
Cannot open a document with no revision or no current revision

### ISSUE THREAD

**Issue by pl281089jig** [avocode/avocode#953]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.8.2
Source UID: 75035
```
Error: Cannot open a document with no revision or no current revision
    at t.openDesign (file:///D:/Avocode-win32-x64/resources/app.asar/dist/avocode.min.js:2:7687)
    at t.openDesignById (file:///D:/Avocode-win32-x64/resources/app.asar/dist/avocode.min.js:2:7375)
    at Constructor._handleEntityOpen (file:///D:/Avocode-win32-x64/resources/app.asar/dist/avocode.min.js:35:19005)
    at Constructor._handleEntityDoubleClick (file:///D:/Avocode-win32-x64/resources/app.asar/dist/avocode.min.js:35:27976)
    at Object.invokeGuardedCallback (D:\Avocode-win32-x64\resources\app.asar\node_modules\react\lib\ReactErrorUtils.js:27:12)
    at executeDispatch (D:\Avocode-win32-x64\resources\app.asar\node_modules\react\lib\EventPluginUtils.js:79:21)
    at Object.executeDispatchesInOrder (D:\Avocode-win32-x64\resources\app.asar\node_modules\react\lib\EventPluginUtils.js:102:5)
    at executeDispatchesAndRelease (D:\Avocode-win32-x64\resources\app.asar\node_modules\react\lib\EventPluginHub.js:43:22)
```

**Comment by connor-bean** [comment]
Hello -

Can you give us more information on what happened? What you were doing when you saw this error?

Thanks!
Connor, Avocode Support

**Comment by petrbrzek** [comment]
Fixed in 2.8.3.
